config/initializers/rack_attack.rb

```diff
@@ -18,7 +18,9 @@ def remote_ip
     # whitelisting). It must implement .increment and .write like
     # ActiveSupport::Cache::Store
 
-    Rack::Attack.cache.store = StoreProxy::RedisStoreProxy.new(Redis.current)
+    Rack::Attack.cache.store = StoreProxy::RedisStoreProxy.new(
+      Redis.new(url: Figaro.env.sidekiq_redis_url)
```

Comment:
Should we add a new key for the DB config of each or just hardcode it in the init files? I kind of prefer the latter since the init files are a form of a config file any how.